test/e2e/framework/pod/create.go

```diff
@@ -247,13 +246,13 @@ func MakeSecPod(podConfig *Config) (*v1.Pod, error) {
 		volumeIndex++
 	}
 
-	podSpec.Spec.Containers[0].VolumeMounts = volumeMounts
-	podSpec.Spec.Containers[0].VolumeDevices = volumeDevices
-	podSpec.Spec.Volumes = volumes
+	podSpec.Containers[0].VolumeMounts = volumeMounts
+	podSpec.Containers[0].VolumeDevices = volumeDevices
+	podSpec.Volumes = volumes
 	if runtime.GOOS != "windows" {
-		podSpec.Spec.SecurityContext.SELinuxOptions = podConfig.SeLinuxLabel
+		podSpec.SecurityContext.SELinuxOptions = podConfig.SeLinuxLabel
```

Comment:
this part should be removed, I think. SecurityContext is already correctly configured in line 216.
Also we should avoid using runtime.GOOS.

Reply:
sounds good, I'll add it to https://github.com/kubernetes/kubernetes/pull/100356